app/views/jobs/_job_details.html.erb

```diff
@@ -1,20 +1,54 @@
-<%= label :job, :resident_name, 'Resdient Name', :class => 'col-form-label'  %>
-<%= job[:FirstName] %>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/divbs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+  
+<form>
```

Comment:
We will have to use `form_tag` because we will have to call API for update action.